.github/workflows/end_to_end_test.yaml

```diff
@@ -12,8 +12,14 @@ jobs:
     steps:
       - name: Echo hello world
         run: echo "hello world"
+      - name: File permission for /usr/local/bin
+        run: ls -ld /usr/local/bin | grep drwxrwxrwx
       - name: Proxy set in /etc/environment
-        run: cat /etc/environment | grep PROXY
+        run: cat /etc/environment | grep HTTP_PROXY
```

Comment:
What if we run some  sort of build process here that requires the proxy?

Reply:
I added a comment noting the Install microk8s would be testing proxy setting.
Including NO_PROXY as well.